core/src/main/java/bisq/core/btc/wallet/Restrictions.java

```diff
@@ -50,16 +53,25 @@ public static Coin getMinTradeAmount() {
         return MIN_TRADE_AMOUNT;
     }
 
-    public static double getDefaultBuyerSecurityDepositAsPercent() {
-        return 0.1; // 10% of trade amount.
+    public static double getDefaultBuyerSecurityDepositAsPercent(PaymentAccount paymentAccount) {
```

Comment:
It's nice to know when nullable is allowed

Reply:
I'll add the Nullable annotation where appropriate.